lib/app.rb

```diff
@@ -50,9 +49,10 @@ class App < Sinatra::Base
     use Routes::Comments
     use Routes::Nits
     use Routes::Teams
+    use Routes::Sessions
```

Comment:
Oh, right. It's checking that the user is logged in, which is not useful for pages that don't need it, like the language info pages (which don't exist yet in this branch).